cmd/controller-manager/app/controller-manager.go

```diff
@@ -94,6 +94,11 @@ member clusters and does the necessary reconciliation`,
 	return cmd
 }
 
+// TODO(sohankunkerkar) Remove when no longer necessary.
+// This Environment variable is a temporary addition to support Red Hat's downstream testing efforts.
+// Its continued existence should not be relied upon.
+const defaultScopeEnv = "DEFAULT_FEDERATION_SCOPE"
```

Comment:
Please define this in the conditional block starting on line 258.  It's not used outside of that block, so defining it up here serves no purpose.

Reply:
Done!